backend/de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/exceptions/DBUniqueConstraintException.java

```diff
@@ -143,4 +122,11 @@ private static String parseConstraintName(String sqlException, String quoteStart
 		return null;
 	}
 
+	@Nullable
+	@Override
+	public String getErrorCode()
+	{
+		return DB_UNIQUE_CONSTRAINT_ERROR_CODE_PREFIX + constraintName;
```

Comment:
hmm, not sure that `constraintName` shall be part of the error code.

Reply:
removed